tests/test_portscan_handler.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import pytest
+import os
+import json
+from lib.portscan_handler import PortScanHandler
+
+
+class TestPortScanHandler():
+    def test_creation(self):
+        port_scan_handler = PortScanHandler()
+        assert type(port_scan_handler) is PortScanHandler
+
+    def test_queue(self):
+        test_event = {"body": '{"target": "ssh.mozilla.com"}'}
+        test_context = None
+        port_scan_handler = PortScanHandler()
+        response = port_scan_handler.queue(test_event, test_context)
```

Comment:
I wonder if this means, any time this test is run, we add the target above to the actual queue (which would actually kick off a scan).

Do we want that? I'm guessing not.

We could have a separate test queue, or maybe include a condition to not add to the queue when the `context` is `None`...or we could use ![moto](https://github.com/spulec/moto) I guess.

Reply:
I will write the new scanner functionality (e.g. TLS observatory) using the same logic and will try to test out `moto` while writing its tests.

Reply:
Nevermind, I figured out how to do this using `moto` while writing necessary classes for TLS Observatory scan. We can use it here as well.
Will make a commit shortly.

Reply:
yep, it's just a dummy queue for testing